### DISCUSSION TITLE
Release Airbyte Platform Docker Images for `armv7l`

### DISCUSSION THREAD

**Discussion by thalesmaoa** [airbytehq/airbyte#32835] [Connector Ideas and Features]
### What area the feature impact?

Airbyte Platform

### Relevant Information

Hi there, is it possible to build a docker for armv7l? 
I have a lot of TimescaleDB (postgreSQL) running in the filed. It has been complicated to log in them individually to check the data. I want to merge them all in the cloud. The problem is that I can't configure an incoming connection. Thus, I need airbyte running locally in each Raspberry and syncing their database to the cloud.

**Comment by marcosmarxm** [comment]
Hello @thalesmaoa thanks for requesting this. Can you rename the request to match a single feature request?
At the moment both aren't in the team roadmap but I kept here for further discussion.

**Reply by thalesmaoa** [reply]
Thanks @marcosmarxm . Just changed as requested.

[a comment by an automated account is not shown]
